swagger.yaml

```diff
@@ -4080,6 +4080,58 @@ paths:
           schema:
             $ref: '#/definitions/error'
 
+  /v2/infra-envs/{infra_env_id}/downloads/minimal-initrd:
```

Comment:
Doesn't need to be in this PR, but do we also need an API for getting a presigned URL?

Reply:
Not sure I understand. This is realistically only going to be used by the image service. And it will use whatever authentication it's provided by the user in the image download url.